.github/CANARY_FAILURE_TEMPLATE.md

```diff
@@ -0,0 +1,5 @@
+---
+title: Canary tests failed
+labels: 'Status: Untriaged'
```

Comment:
Can we add the tests label here as well?

Reply:
Good input 3ba4f387dcae0ca34cf2991ecb6b56098a64beda